<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player1_Chooser</class>
 <widget class="QWidget" name="Player1_Chooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../assets/assets.qrc">:/Items/titlescreen_new.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="ChooserBg">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>175</y>
     <width>360</width>
     <height>325</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../assets/assets.qrc">:/Items/SeedChooser_Background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose Initiate Player1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QStackedWidget" name="Stacked_SnowPea">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="ChooseButton_SnowPea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/ShovelBank.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Img_SnowPea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>3</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/Plants/SnowPea.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="SnowPea_Chosen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/ui/SeedChooser_Button_Glow.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="BackButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QStackedWidget" name="Stacked_PeaShooterSingle">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="ChooseButton_PeaShootSingle">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/ShovelBank.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Img_PeaShooterSingle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>3</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/Plants/PeaShooterSingle.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="PeaShooterSingle_Chosen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/ui/SeedChooser_Button_Glow.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="NextBotton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QStackedWidget" name="Stacked_Cabbagepult">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="ChooseButton_Cabbagepult">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/ShovelBank.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Img_Cabbagepult">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>146</width>
          <height>146</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/Plants/Cabbagepult.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="Cabbagepult_Chosen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/ui/SeedChooser_Button_Glow.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QStackedWidget" name="Stacked_Chomper">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QPushButton" name="ChooseButton_Chomper">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/ShovelBank.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Img_Chomper">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>3</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>81</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/Plants/Chomper.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="Chomper_Chosen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../../assets/assets.qrc">:/Items/ui/SeedChooser_Button_Glow.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
